Automattic/wp-calypso
# Inline Help: move selected result state from Redux to local

Followup to #55833 that removes the `selectedResult` state from Redux and moves it to local state of the `InlineHelpPopover` component.

The `InlineHelpSearchResults` component no longer dispatches the `selectResult` action, but the result selection is integrated into the `onSelect` callback prop that is passed to it from the `Popover` component.

The `PopUpSearch` component by @jdc91 binds the `selectResult` action, but doesn't use it. The component is unfinished, so it might need to implement similar local state if it wants to show the "contextual" type of results.

**How to test:**
Exactly the same steps as in #55833:
- open inline help popover
- select one of the default (contextual) results
- verify that a secondary view for this result is displayed:

<img width="324" alt="Screenshot 2021-08-30 at 15 52 05" src="https://user-images.githubusercontent.com/664258/131471678-f9a0c4c8-3bea-4bd3-a9f4-1a8bece105dd.png">
